### DISCUSSION TITLE
Possibility for PeerTube support?

### DISCUSSION THREAD

**Discussion by TeknikalDomain** [Lednerb/bilberry-hugo-theme#328] [Ideas]
Since it seems that video hosting sites are still being added to the supported providers, how about seeing if it's *possible* to slot [PeerTube](https://joinpeertube.org) in as well?

PeerTube is a federated network, similar to Mastodon or Matrix, meaning there is no one central PeerTube hosting site, so that would need to be accounted for, however, in my personal fork (where I've kept my own tweaks and site-specifc additions) I've already got 99% of the implementation done, I'd just need to generalize it to not be specific to my instance or my site's configuration. I could probably pull a PR up for that in just a few hours, once I find a gap in my schedule.

**Comment by igor-baiborodine** [comment]
Sure, why not.

**Comment by TeknikalDomain** [comment]
PR #330 has been created for this.
